### PR TITLE
Handle new response of relayed V3

### PR DESCRIPTION
**Pull request by andreibancioiu**
This isn't a breaking change per-se, since Relayed V3 isn't active on Mainnet. Additionally, the previous deployment on Devnet contained a few issues.

Related:
 - https://github.com/multiversx/mx-chain-proxy-go/pull/465
 - https://github.com/multiversx/mx-chain-go/pull/6529

If one wants to retrieve a Relayed (V3) with its inner transactions, she should simply do:

```
const transaction = networkProvider.getTransaction(hashOfParentTransaction);
console.log(transaction.innerTransactions);
```

Alternatively, if one wants to retrieve only an inner transaction, she must specify both the hash of the inner transaction, and the hash of the parent transaction:

```
// API is somehow cumbersome, kept this way to avoid breaking changes.
const transaction = networkProvider.getTransaction(hashOfInnerTransaction, false, hashOfParentTransaction);
```

### CODE REVIEW

**Review by danielailie — APPROVED** [review]

File: src/proxyNetworkProvider.ts [hunk 1]
```diff
@@ -103,20 +103,30 @@ export class ProxyNetworkProvider implements INetworkProvider {
         return tokenData;
     }
 
-    async getTransaction(txHash: string, withProcessStatus?: boolean): Promise<TransactionOnNetwork> {
+    async getTransaction(
+        txHash: string,
+        withProcessStatus?: boolean,
+        relayedTxHash?: string,
```
[the remark this answers is not in the document]
Comment (popenta) [reply src/proxyNetworkProvider.ts:109]: So one has to provide both the relayed txHash and the inner transaction hash? Can't simply get transaction by inner transaction hash?

**Review by popenta — APPROVED** [review]

File: src/proxyNetworkProvider.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreibancioiu) [reply src/proxyNetworkProvider.ts:109]: That is true.

However, one can also fetch the parent (big) transaction directly, then access `innerTransactions[n]`.

[the remark this answers is not in the document]
Comment (andreibancioiu) [reply src/proxyNetworkProvider.ts:109]: Function signature is somehow cumbersome. Kept this way to avoid breaking changes.

**Comment by danielailie** [comment]
This is not needed anymore

**Pull request closed without merging**